### ISSUE TITLE
Improve diet tagging

### ISSUE THREAD

**Issue by jolesh** [Zverik/every_door#496]
Diet tagging has three possible values: yes, no, only

Maybe make a small table with the diet forms on the left and the values selectable, like this:
Diets:
| vegetarian | no | yes | only |
| vegan | no | yes | only |
| halal | no | yes | only |
...

Maybe accessible through a menu, just like opening hours opens a dialog.

**Comment by Robot8A** [comment]
I support this. Checking the code, it should require either adding a new ComboType, or changing the current ComboType.multi behaviour.

**Comment by Zverik** [comment]
Hmm. It looks like I did the `multiCombo` fields wrong, not even supporting "`no`" values. The iD editor does it right though.

One question: can multiple diet options have the `only` value?

**Comment by Robot8A** [comment]
Yes, one restaurant could be both for example only gluten-free and only vegetarian.

**Comment by patepelo** [comment]
It would be similar to "Take away only" field. 

Maybe we can take the code from there and reuse it?
